packages/core-transaction-pool/__tests__/interface.test.js

```diff
@@ -218,4 +218,10 @@ describe('Transaction Pool Interface', () => {
       expect(poolInterface.checkDynamicFeeMatch).toBeFunction()
     })
   })
+
+  describe('checkIfSenderHasVoteTransactions', () => {
+    it('should be a function', () => {
+      expect(poolInterface.checkIfSenderHasVoteTransactions).toBeFunction()
+    })
+  })
```

Comment:
This looks like a duplicate with what is in `packages/core-transaction-pool-mem/__tests__/connection.test.js`:

```
401   describe('checkIfSenderHasVoteTransactions', () => {
402     it('should be a function', () => {
403       expect(connection.checkIfSenderHasVoteTransactions).toBeFunction()
404     })
...
```

Not sure how tests are organized and what are the best practices but I think one of the two can/should be removed.

Reply:
the interface test should check for method signature and throw when called. 

as for connection.test.js it will test then actual implementation of the pool-mem.